db.py

```diff
@@ -0,0 +1,277 @@
+import psycopg2
+from datetime import datetime
+
+
+def get_connection_to_database():
+    conn = psycopg2.connect(
+        dbname="",
+        user="",
+        password='', #ВСТАВИТЬ ДАННЫЕ
+        host='',
+        port=,
+    )
+    return conn
+
+
+def decorator_add_connection(func):
+    def wrapper(*args, **kwargs):
+        conn = get_connection_to_database()
+        result = func(*args, **kwargs, conn=conn)
+        conn.close()
+        return result
+
+    return wrapper
+
+
+@decorator_add_connection
+def create_database(conn=None):  # Создаем базу данных
+    conn.autocommit = False
+    cur = conn.cursor()
+    cur.execute("""CREATE TABLE IF NOT EXISTS public.question(
+    id integer NOT NULL,
+    question_text character varying(100) NOT NULL,
+    publish_date time without time zone,
+    PRIMARY KEY (id)
+    );""")
+    conn.commit()
+    cur.execute("""CREATE TABLE IF NOT EXISTS public.choice (
+    id integer NOT NULL,
+    choice_text character varying(30) NOT NULL,
+    votes bigint,
+    question_id integer NOT NULL,
+    PRIMARY KEY (id),
+    CONSTRAINT question_id FOREIGN KEY (question_id)
+        REFERENCES public.question (id) MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+        NOT VALID
+    );""")
+    conn.commit()
+    cur.execute("""CREATE TABLE IF NOT EXISTS public.statistic (
+    id bigint NOT NULL,
+    tg_user_id integer NOT NULL,
+    question_id integer NOT NULL,
+    choice_id integer NOT NULL,
+    CONSTRAINT choice_id FOREIGN KEY (choice_id)
+        REFERENCES public.choice (id) MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+        NOT VALID,
+    CONSTRAINT question_id FOREIGN KEY (question_id)
+        REFERENCES public.question (id) MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+        NOT VALID
+    );""")
+    conn.commit()
+    try:  # Создаем самый важный вопрос :)
+        publish_date = datetime.now()
+        cur.execute("INSERT INTO question VALUES (1, 'Вам нравится мой бот?', (%s));", (publish_date,))
+        cur.execute("INSERT INTO choice VALUES (1, 'Да', 0, 1),(2, 'Да, очень', 0, 1),(3, 'Определенно да', 0, 1);")
+    except Exception:
+        cur.close()
+    else:
+        conn.commit()
+        cur.close()
+
+
+@decorator_add_connection
+def get_all_questions(conn=None):  # Получаем все вопросы с ответами
+    cur = conn.cursor()
+    cur.execute('SELECT * FROM question ORDER BY id ASC')
+    select = cur.fetchall()
+    if not select:
+        return ['Повелитель, Вы забыли? Вопросов еще нет', False]  # Если вопросов нет, возвращаем список с False
+    select_list = [list(b) for b in select]
+    answer = 'Вопросы:\n'
+    for i in select_list:
+        answer += f'Вопрос {i[0]}: {i[1]}\n'
```

Comment:
https://github.com/thehighestmath/umschool-final-task/blob/master/review_comments.md#2-%D0%BA%D0%BE%D0%BD%D0%BA%D0%B0%D1%82%D0%B5%D0%BD%D0%B0%D1%86%D0%B8%D1%8F-%D1%81%D1%82%D1%80%D0%BE%D0%BA-%D0%B2-%D1%86%D0%B8%D0%BA%D0%BB%D0%B5---%D1%8D%D1%82%D0%BE-%D0%B7%D0%BB%D0%BE